README.md

```diff
@@ -88,6 +88,7 @@ $ sudo dnf copr enable -y nsoffer/ovirt-imageio-preview
 
 Install additional packages for Fedora, CentOS, and RHEL:
 ```shell
+$ cd vdsm
 $ sudo dnf install -y `cat automation/check-patch.packages`
```

Comment:
I'm not sure the "cd vdsm" part is needed.

The automation directory should be removed and this command should be replaced. I think Albert is working this in another PR.